modules/engage-theodul-plugin-tab-transcript/src/main/resources/static/main.js

```diff
@@ -214,35 +214,9 @@ define(["jquery", "underscore", "backbone", "engage/core"], function($, _, Backb
         }
     });
 
-    function requestTranscriptWorkflow() {
```

Comment:
Leave the function here - add a comment saying that it is currently not used/disabled for now.